app/code/Magento/Catalog/Model/Product/Option/Type/DefaultType.php

```diff
@@ -392,14 +392,14 @@ public function getProductOptions()
     }
 
     /**
-     * Return final chargable price for option
+     * Return final chargeable price for option
      *
      * @param float $price Price of option
      * @param boolean $isPercent Price type - percent or fixed
      * @param float $basePrice For percent price type
      * @return float
      */
-    protected function _getChargableOptionPrice($price, $isPercent, $basePrice)
+    protected function _getChargeableOptionPrice($price, $isPercent, $basePrice)
```

Comment:
Protected methods cannot be removed, please check https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html#php:
> Public and protected method removal 
> Mark the method with the `@deprecated` tag instead of removing it.

Reply:
?